sqrl-math/README.md

```diff
@@ -0,0 +1,168 @@
+# SQRL Mathematical Functions
+
+The `com.datasqrl.math` package provides a collection of advanced mathematical functions for use within the SQRL framework.
+
+## Compile and Run
+
+First, ensure that all dependencies are installed by running the Maven package command:
+```bash
+mvn package
+```
+
+Once the build is complete, compile your SQRL script along with the schema:
+```bash
+sqrl compile math.sqrl schema.graphqls
+```
+
+Start the containers with:
+```bash
+docker compose up --build
+```
+
+## Functions Overview
+
+Each function operates on DOUBLE (denoted as d) and BIGINT (denoted as b) in the examples.
+You can use these UDFs in your SQRL scripts to perform advanced mathematical and statistical computations.
+
+### Atan2
+
+**Computes the arc tangent of y/x in a way that preserves the quadrant.**
```

Comment:
Please create a table that lists the functions with name in one column and a description in the other, i.e. similar to https://www.datasqrl.com/docs/reference/sqrl/functions/time/.

That's easier to read that having to go down a list. Provide one example of how to use the function - the user can generalize from there.